cli/tests/integrations/test_package.py

```diff
@@ -265,20 +308,6 @@ def test_describe_app_cli():
         stdout=stdout)
 
 
-def test_describe_specific_version():
-    stdout = file_bytes(
-        'tests/data/package/json/test_describe_marathon_0.11.1.json')
-
-    returncode_, stdout_, stderr_ = exec_command(
-        ['dcos', 'package', 'describe', '--package-version=0.11.1',
-         'marathon'])
-
-    assert returncode_ == 0
-    output = json.loads(stdout_.decode('utf-8'))
-    assert output == json.loads(stdout.decode('utf-8'))
-    assert stderr_ == b''
-
-
```

Comment:
Good catch. This was a duplicate test. `test_describe_package_version` tests the same thing.